src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php

```diff
@@ -89,7 +89,14 @@ protected function findTemplate($template, $throw = true)
         }
 
         if (false === $file || null === $file) {
-            throw new \Twig_Error_Loader(sprintf('Unable to find template "%s".', $logicalName), -1, null, $previous);
+            try {
+                list($namespace, $name) = $this->parseName($logicalName);
```

Comment:
Maybe the exception's message what we catch [here](https://github.com/symfony/symfony/blob/2.3/src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php#L80) should be thrown, and [this](https://github.com/symfony/symfony/blob/2.3/src/Symfony/Bundle/TwigBundle/Loader/FilesystemLoader.php#L87) should be the ```$previous```